src/components/nxtp/SwapXpollinate.tsx

```diff
@@ -997,9 +1016,9 @@ const SwapXpollinate = ({
             />
           )}
         </Row>
-
+        <Row className="top-dropdown" justify={"center"}>
         <Collapse activeKey={activeKeyTransactions} accordion ghost>
-
+        <Collapse style={{ marginTop: 24 }} accordion ghost>
```

Comment:
This breaks the functionality of the Collapse to toggle sections using set `activeKeyTransactions`.